src/mlpack/methods/reinforcement_learning/gae.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file policygrad.hpp
```

Comment:
@zoq , I have changed this to gae.hpp. Gae stands for generalised advantage estimation. Please provide your inputs

Reply:
Sorry for the slow response on this one, this sounds fine for me.

Reply:
Hello @zoq . You might be busy because of GSOC Queries.
However I would like you to have a look of this pull request once you get free.
I have added two of the 6 policy gradient methods as described in the preliminaries of paper.(https://arxiv.org/pdf/1506.02438.pdf) (1 and 2)
 However I am having some difficulties in writing the test case for policy gradients.
You may like to add the codes to the software.